css/src/components/timeline.scss

```diff
@@ -0,0 +1,51 @@
+$timeline-content-font-size: $font-size-8 !default;
+$timeline-timestamp-font-size: $font-size-9 !default;
+$timeline-timestamp-font-weight: $weight-semilight !default;
+$timeline-description-padding-top: $spacer-4 !default;
+$timeline-description-line-height: $line-height-normal;
+
+$timeline-item-border-left: $border-width solid $border !default;
+$timeline-item-min-height: $layout-6 !default;
+$timeline-item-padding-bottom: $spacer-7 !default;
+$timeline-item-padding-left: $spacer-8 !default;
+
+$timeline-badge-top-offset: -0.35em !default;
```

Comment:
What's the reasoning behind these values? I see they're used for put the badge on the border, but I'm curious as to the reason these specific `em` values are chosen.

Reply:
Just so I'm trying to be constructive, here is a transform I was able to reason through. You don't have to go with this but I think it achieves something very similar to what you have:

```scss
transform: translate(
		calc(-#{$timeline-item-padding-left} - 50%), // $spacer-8/$timeline-item-padding-left is the space reserved for the line on the side.
		-#{math.div($timeline-timestamp-font-size, 2)} // half the font size of the timestamp line
);
```

Reply:
You do need to keep position absolute though in order to ensure there's no space reserved for the badges.

Reply:
Fixed via conversation